modules/xmpp/strophe.jingle.js

```diff
@@ -255,40 +249,6 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
             this.eventEmitter.emit(XMPPEvents.CALL_ENDED, sess, reasonCondition, reasonText);
             break;
         }
-        case 'transport-replace': {
```

Comment:
Suggest adding a log that transport-replace in not supported, just to help people debug if it happens to be used somehow